tools/shared/src/main/scala/org/scalajs/core/tools/javascript/Printers.scala

```diff
@@ -29,17 +28,21 @@ import org.scalajs.core.tools.sourcemap.SourceMapWriter
 object Printers {
```

Comment:
We should drop a comment at top of this file explaining that it is written with raw performance in mind, and hand-optimized, so that our future selves don't try to "clean it up" by factoring out stuff ;-)